graphql_compiler/tests/schema_transformation_tests/test_rename_schema.py

```diff
@@ -183,6 +240,156 @@ def test_interface_rename(self):
             {"NewKid": "Kid", "NewCharacter": "Character"}, renamed_schema.reverse_name_map
         )
 
+    # def test_suppress_interface_implementation(self):
```

Comment:
commented out because not sure what to do with interfaces, see discussion in PR 859